modules/ROOT/pages/index.adoc

```diff
@@ -1,81 +1,75 @@
 [[index]]
+:description: This section describes the Neo4j GraphQL Library.
 = Neo4j GraphQL Library
-:experimental:
-:chapter-label:
-:toc-title: Contents
-//:front-cover-image: image::title-page.png[]
-:header-title: NEO4J GRAPHQL LIBRARY
-:title-page-background-image: image::title-page.png[]
 
-The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform and mobile applications by tapping into the power of connected data.
+[NOTE]
+====
+The GRANDstack starter app has been deprecated. 
+For more information, read the section on xref::deprecations.adoc[Deprecations].
+====
 
-With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have application data treated as a graph natively from the front-end all the way to storage, avoiding duplicate schema work and ensuring flawless integration between front-end and backend developers.
+The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform, and mobile applications by tapping into the power of connected data.
 
-Written in TypeScript, the library's schema-first paradigm lets developers focus on the application data they need, while taking care of the heavy lifting involved in building the API.
+With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have data treated as a graph natively from the front-end all the way to storage.
+This avoids duplicate schema work and ensures flawless integration between front-end and backend developers.
 
-TIP: Are you a GRANDstack user? We have deprecated the GRANDstack starter app, find more information xref::deprecations.adoc[here].
+== How it works
 
-== How does it work?
+Written in TypeScript, the Neo4j GraphQL Library offers a set of type definitions that describes the shape of your graph data.
```

Comment:
I would remove the language, and highlight that the library requires type definitions of the user to generate the schema.

```suggestion
The Neo4j GraphQL Library requires a set of type definitions that describes the shape of your graph data.
```